content/en/docs/concepts/storage/storage-capacity.md

```diff
@@ -103,14 +102,12 @@ to handle this automatically.
 
 ## Enabling storage capacity tracking
 
-Storage capacity tracking is a beta feature and enabled by default in
-a Kubernetes cluster since Kubernetes 1.21. In addition to having the
-feature enabled in the cluster, a CSI driver also has to support
+Storage capacity tracking is a stable feature and always enabled in a
+Kubernetes cluster since Kubernetes 1.24. A CSI driver also has to support
 it. Please refer to the driver's documentation for details.
```

Comment:
```suggestion
Kubernetes v{{< skew currentVersion >}} includes cluster-level API support for storage
capacity tracking. To use this you must also be using a CSI driver that supports capacity
tracking. Consult the documentation for the CSI drivers that you use to find out whether
this support is available and, if so, how to use it.
If you are not running Kubernetes v{{< skew currentVersion >}}, check the documentation
for that version of Kubernetes.
```

I would however also move this particular section to the _start_ of the page (just before `<!-- body -->`) and change the heading to read:

```markdown
## {{% heading "prerequisites" %}}
```

You can then also reword or remove:
```markdown
Tracking storage capacity is supported for {{< glossary_tooltip
text="Container Storage Interface" term_id="csi" >}} (CSI) drivers and
[needs to be enabled](#enabling-storage-capacity-tracking) when installing a CSI driver.
```